modules/library/src/main/scala/zio/elasticsearch/ElasticRequest.scala

```diff
@@ -552,45 +556,59 @@ object ElasticRequest {
       extends ElasticRequest[SearchResult]
       with HasFrom[SearchRequest]
       with HasRouting[SearchRequest]
+      with HasSize[SearchRequest]
       with HasSort[SearchRequest]
-      with HasSize[SearchRequest] {
+      with HasSourceFiltering[SearchRequest] {
     def aggregate(aggregation: ElasticAggregation): SearchAndAggregateRequest
 
+    def excludes(fields: String*): SearchRequest
```

Comment:
Do we want to support `excludes(field: String, fields: String*)`?

Reply:
I think we'll have to choose one or the other as having both will probably cause `ambiguity` as `excludes(oneValue)` satisfies both. I think the decision should come down to whether we think the list will be used in a dynamic manner, in which case the stricter form can be less convenient.